bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_tau_NLO/HeavyNeutrino_trilepton_mNgtmWltmZ_V-0p01_tau_NLO/HeavyNeutrino_trilepton_V-0p01_tau_NLO_customizecards.dat

```diff
@@ -1,4 +1,4 @@
-set param_card mass 9900012 1000
+set param_card mass 9900012 HNLMASS
```

Comment:
What is the width in this case? Is the model default ok?

Reply:
Thank you for the checks! In the last line of the same file the width is recomputed for each mass point so it should be ok.

Reply:
I see. Is only displayed when clicking on view file. Ok.